src/app/core/mpi/components/botones-registro.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Output, EventEmitter } from '@angular/core';
+
+@Component({
+    selector: 'botones-registro',
```

Comment:
```suggestion
    selector: 'mpi-botones-registro',
```

Agregar el prefijo MPI